tmt/steps/report/reportportal.py

```diff
@@ -49,6 +51,16 @@ class ReportReportPortalData(tmt.steps.report.ReportStepData):
         metavar="NAME",
         default=None,
         help="The launch name (base name of run id used by default).")
+    ca_cert_file: Optional[str] = tmt.utils.field(
+        option="--ca-cert-file",
+        metavar="CA_CERT",
```

Comment:
`PATH` or `FILE` would be better - `CA_CERT` is slightly ambiguous - shall I put a file path here, or the actual certificate? `--ca-cert-file PATH` would leave no space for doubt.

Reply:
ACK

Reply:
@happz 

As per @Natalia Bubakova
We could use this in the bashrc, as follow
```
export REQUESTS_CA_BUNDLE=<PATH to file>
```

Do you thing this option needed?

Reply:
That is correct, `REQUESTS_CA_BUNDLE` should work. If you're fine with setting this envvar - which might help other places where `requests` are used in tmt code, not just the RP plugin - then you don't need the option. Sounds like an acceptable solution to me.